src/main/java/core/basesyntax/ArrayListIndexOutOfBoundsException.java

```diff
@@ -1,7 +1,7 @@
 package core.basesyntax;
 
 public class ArrayListIndexOutOfBoundsException extends RuntimeException {
-    public ArrayListIndexOutOfBoundsException(String message) {
-        super(message);
+    public ArrayListIndexOutOfBoundsException(int index) {
+        super(String.format("Index %d is out of array list bounds.", index));
```

Comment:
return previous version please) pass this message when creating an instance of the exception